tests/test-modules-export-export.php

```diff
@@ -397,7 +397,7 @@ public function test_sanityChecks( $module, $prerequisite ) {
 			}
 			if ( strpos( $format, '\HTMLBook\HTMLBook' ) !== false ) {
 				// TODO: HTMLBook is too strict we don't pass the validation
-			} elseif ( $runtime->isPHPDBG() && strpos( $format, '\Epub\Epub' ) !== false ) {
```

Comment:
PHPUnit > 8 does not require this check, this code is always running in testing thread